app/helpers/earthworks_geoblacklight_helper.rb

```diff
@@ -1,9 +1,8 @@
 module EarthworksGeoblacklightHelper
   include GeoblacklightHelper
 
-  def document_available?
-    (@document.public? && @document.available?) ||
-      (@document.same_institution? && user_signed_in? && @document.available?)
+  def document_available?(document = @document)
+    super && document.available?
   end
```

Comment:
because of other changes made in `item_map_viewer_component`...see https://stanfordlib.slack.com/archives/C0AAK1HNY/p1720731983010169